api/app/main.py

```diff
@@ -145,7 +147,7 @@ async def get_noon_forecasts_summaries(request: schemas.StationCodeList, _: bool
     """ Returns summaries of noon forecasts for given weather stations """
     try:
         LOGGER.info('/noon_forecasts/summaries/')
-        now = datetime.datetime.now(tz=datetime.timezone.utc)
+        now = time_utils.get_utc_now()
```

Comment:
I don't think that the fixtures pick it up correctly when you do it this way.
:(
I think have to import:
```import app.time_utils```
and then use:
```app.time_utils.get_utc_now()```